pkg/daemon/update.go

```diff
@@ -495,7 +523,7 @@ func (dn *Daemon) reboot(rationale string) error {
 	if (dn.recorder != nil) {
 		dn.recorder.Eventf(&corev1.Node{ObjectMeta: metav1.ObjectMeta{Name: dn.name}}, corev1.EventTypeNormal, "Reboot", rationale)
 	}
-	glog.Infof("Rebooting: %s", rationale)
```

Comment:
Does this remove the "Rebooting" message from the normal Daemon log output?

Reply:
Nope, we still call `glog.Info(message)` at the top of `logSystem()`

Reply:
thanks!